frontend/controller/actions/identity.js

```diff
@@ -680,6 +681,142 @@ export default (sbp('sbp/selectors/register', {
   'gi.actions/identity/logout': (...params) => {
     return sbp('okTurtles.eventQueue/queueEvent', 'ACTIONS-LOGIN', ['gi.actions/identity/_private/logout', ...params])
   },
+  'gi.actions/identity/changePassword': async ({
+    identityContractID,
+    username,
+    oldIPK,
+    oldIEK,
+    newIPK: IPK,
+    newIEK: IEK,
+    newSAK: SAK,
```

Comment:
Are these flowtype annotations...? I thought they weren't allowed like this and had to be done after the `}` as in `}: { newIPK: <type>`?

Reply:
No, they're destructuring names.

```js
const foo = { bar: 1 }
const { bar: baz } = foo
console.log(baz) // 1
console.log(typeof bar) // undefined
```

Reply:
🤦 

Thanks!